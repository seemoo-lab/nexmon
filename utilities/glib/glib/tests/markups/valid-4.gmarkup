<foo>
<bar a='1'/>
<bar a="2"/>
<bar a='3"'/>
<bar a="4'"/>
<bar a="5''''"/>
</foo>

