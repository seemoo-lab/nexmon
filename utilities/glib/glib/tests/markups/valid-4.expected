ELEMENT 'foo'
  TEXT '
'
  ELEMENT 'bar'
  a="1"
  END 'bar'
  TEXT '
'
  ELEMENT 'bar'
  a="2"
  END 'bar'
  TEXT '
'
  ELEMENT 'bar'
  a="3""
  END 'bar'
  TEXT '
'
  ELEMENT 'bar'
  a="4'"
  END 'bar'
  TEXT '
'
  ELEMENT 'bar'
  a="5''''"
  END 'bar'
  TEXT '
'
END 'foo'
